packages/picasso-rich-text-editor/src/plugins/CodePlugin/CodePlugin.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+import type { RTEPlugin } from '../api'
+import { RTEPluginMeta, Toolbar } from '../api'
+import CodeButton from './CodeButton'
+
+const PLUGIN_NAME = 'code'
+
+export type Props = {
+  testIds?: {
+    button?: string
+  }
+}
+
+const EmojiPlugin: RTEPlugin<Props> = ({ testIds }) => {
```

Comment:
I think this is a leftover
```suggestion
const CodePlugin: RTEPlugin<Props> = ({ testIds }) => {
```

Reply:
done